cves/CVE-2009-2693.yml

```diff
@@ -209,10 +231,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer: The mistake that was made here was simple. The developer neglected to consider
-    that someone might try to navigate outside of the root directory and never
-    put in test cases and exceptions for when that happens. Since there was a lack
-    of unit testing, the directory traversal was never tested for and stayed in the 
-    code for a while unnoticed. This isn't exactly a mistake but more of an oversight 
-    on the developers part. The vulnerability was fixed by adding checks for "../" 
-    for any path that was being constructed. 
+  answer: It was a very simple mistake that was made. The developer did not take 

```

Comment:
Try to make this sentence flow a bit better by getting rid of the passive voice.  Maybe:

_The mistake made was very simple._

Reply:
Fixed, thanks